.github/workflows/build-images.yml

```diff
@@ -288,7 +288,7 @@ jobs:
         default: ghcr.io/${{ env.OWNER_LC }}/vertica-k8s:${{ github.sha }}-v2
         conditionals-with-values: |
           ${{ inputs.v2_vertica_image != '' }} => ${{ inputs.v2_vertica_image }}
-          ${{ github.event_name == 'pull_request' }} => vertica-k8s:kind-v2
+          ${{ github.event_name == 'pull_request' }} => verticadocker/vertica-k8s-private:latest-master
```

Comment:
We will use this image name when we build the image a few steps down. We need that build step to use some other image name. Otherwise, it will overwrite the image we want to use. Also, this doesn't handle the case when we push to main. That will use the ghcr.io image name from line 288.

Reply:
Agree! I aslo think about how to push the main. Will fix it as your suggestion!